wolfcrypt/src/port/Espressif/esp32_util.c

```diff
@@ -0,0 +1,62 @@
+/**
+ * util.c
```

Comment:
Please change to march filename and standard:

`/*esp32_utils.c`

Reply:
Yes. I changed all files to match filename and standard.